examples/gno.land/p/demo/acl/acl.gno

```diff
@@ -92,11 +118,139 @@ func (d *Directory) AddUserToGroup(user std.Address, group string) {
 	if ok {
 		groups = existing.([]string)
 	}
-	groups = append(groups, group)
+	groups = appendGroupsWithoutDuplicates(groups, []string{group})
+	d.userGroups.Set(user.String(), groups)
+}
+
+func (d *Directory) AddUserToGroups(user std.Address, newGroups []string) {
+	existing, ok := d.userGroups.Get(user.String())
+	var groups []string
+	if ok {
+		groups = existing.([]string)
+	}
+	groups = appendGroupsWithoutDuplicates(groups, newGroups)
 	d.userGroups.Set(user.String(), groups)
 }
 
-// TODO: helpers to remove permissions.
-// TODO: helpers to adds multiple permissions at once -> {verbs: []string{"read","write"}}.
-// TODO: helpers to delete users from gorups.
-// TODO: helpers to quickly reset states.
+func (d *Directory) RemoveUserFromGroup(user std.Address, group string) {
```

Comment:
Should we return an error or a boolean indicating success when removing a field?